### ISSUE TITLE
[SCREENREADER]: JAWS is reading from the top of the screen on GIBCT after selecting a school

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#1062]
Migrated from -- [vets.gov issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15383).
## Issue

After you select a school, when tabbing past Scholarships to the next field “Enrolled” the screen reader (JAWS) starts reading from the top of the frame “hide eligibility details, what is your military status?...” this continues as you tab down the screen JAWs starts reading from the top with every tab.

## Acceptance Criteria

As a screen reader user, I only want to hear the label and select menu choices that are pertinent to me (currently in focus). I do not want to hear the entire form read again.

## Environment

* Windows
* IE11
* JAWS

## Possible Fix
I believe this could have to do with the select menus being nested in multiple layers of divs in some cases. The issue starts with the `Enrolled` dropdown, which is the first one that has a nesting level. Screenshots attached below.

---
![screen shot 2018-11-30 at 12 40 44 pm](https://user-images.githubusercontent.com/934879/49308346-a0d29c00-f49d-11e8-9415-ad97a7400116.png)

---
![screen shot 2018-11-30 at 12 40 58 pm](https://user-images.githubusercontent.com/934879/49308353-a5975000-f49d-11e8-819b-6bceabf430b7.png)
 
## PR
https://github.com/department-of-veterans-affairs/vets-website/pull/11518

**Comment by peterhill-oddball** [comment]
Quote of Rian Fowler's comment in the old issue:

This nesting occurs when the nontraditional value is set in "Enrolled." I think the intent was to have these selects grouped with the Enrolled question because they're dependent on it but maybe this was done incorrectly. 

Should we try to use fieldsets to make this grouping more logical? 

![screen shot 2019-01-14 at 14 14 13](https://user-images.githubusercontent.com/4998130/51141535-c8d84d80-1806-11e9-87e8-c8cddd6b484c.png)

**Comment by 1Copenut** [comment]
@desireeturner I'm going through our 508 issue backlog and re-assigning these for GIBCT. Please let me know if they should go to another person or group. Thank you.

**Comment by hughes-dustin** [comment]
branch: bah-1062
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/11439

**Comment by delli-gatti-michael** [comment]
The "Enrolled" issue described above has been fixed on the `bah-1062-b` branch, but the fix has introduced a new issue where after tabbing to the "Hide calculator fields" button results in JAWS reading the Tuitions and fees, Scholarships, and Enrolled fields

**Comment by delli-gatti-michael** [comment]
local testing complete and PR approved

**Comment by delli-gatti-michael** [comment]
Fix is now in staging

**Comment by desireeturner** [comment]
@pjhill This issue is live on staging and ready for your review. Let us know once you've had a chance to review if this issue can be closed out. Thank you!

**Comment by pjhill** [comment]
I am unable to reproduce the issues in staging with Win10 + IE11 + NVDA.  That said, unless I am misunderstanding about the steps to reproduce, I am also unable to reproduce the issue with Win10 + IE11 + NVDA in production as of this morning.  I do not have access to a JAWS license.

Steps performed to reproduce -- 

1. Navigate to [GI Bill Comparison Tool in staging](https://staging.va.gov/gi-bill-comparison-tool/).
1. Leave default selections for all fields on first page.
1. Entered a City ("chesapeake") into the "downshift-input-2" input
1. Selected "Search Schools"
1. Selected the first school "AVIATION INSTITUTE OF MAINTENANCE-CHESAPEAKE"
1. Tabbed through elements to the "School Calendar" drop down
1. Selected "Non-Traditional" from the drop down
1. Tabbed back up to "Scholarships"
1. Tabbed down into the Enrolled box in order to get the screen reader to announce

Result: NVDA announced "Enrolled, learn more, combo box collapsed"

@1Copenut You'll need to take a look at this issue with JAWS at any rate, but I believe you were the originator of the issue and I simply migrated it.

**Comment by 1Copenut** [comment]
@desireeturner I tested with JAWS on IE11 and Chrome. The extra information is not being read out, so I'll move this to close.
